### PR TITLE
add comment help info to diff help dialog

### PR DESCRIPTION
**Pull request by meganrogge**
cc @alexr00

### CODE REVIEW

File: src/vs/workbench/contrib/codeEditor/browser/diffEditorHelper.ts [hunk 1]
```diff
@@ -88,6 +89,7 @@ function createScreenReaderHelp(): IDisposable {
 		const editorService = accessor.get(IEditorService);
 		const codeEditorService = accessor.get(ICodeEditorService);
 		const keybindingService = accessor.get(IKeybindingService);
```
Comment (meganrogge) [inline src/vs/workbench/contrib/codeEditor/browser/diffEditorHelper.ts:91]: test

File: src/vs/workbench/contrib/codeEditor/browser/diffEditorHelper.ts [hunk 2]
```diff
@@ -88,6 +89,7 @@ function createScreenReaderHelp(): IDisposable {
...
+		const contextKeyService = accessor.get(IContextKeyService);
 
 		const next = keybindingService.lookupKeybinding(AccessibleDiffViewerNext.id)?.getAriaLabel();
 		const previous = keybindingService.lookupKeybinding(AccessibleDiffViewerPrev.id)?.getAriaLabel();
```
Comment (meganrogge) [inline src/vs/workbench/contrib/codeEditor/browser/diffEditorHelper.ts:95]: test 2

**Review by alexr00 — APPROVED** [review]

**Review by hediet — APPROVED** [review]

File: src/vs/workbench/contrib/accessibility/browser/accessibilityContributions.ts [hunk 3]
```diff
@@ -74,14 +74,6 @@ class AccessibilityHelpProvider implements IAccessibleContentProvider {
 	) {
 	}
```
Comment (meganrogge) [inline src/vs/workbench/contrib/accessibility/browser/accessibilityContributions.ts:75]: test

File: src/vs/workbench/contrib/accessibility/browser/accessibilityContributions.ts [hunk 4]
```diff
@@ -100,31 +92,47 @@ class AccessibilityHelpProvider implements IAccessibleContentProvider {
...
 		if (options.get(EditorOption.tabFocusMode)) {
-			content.push(this._descriptionForCommand(ToggleTabFocusModeAction.ID, AccessibilityHelpNLS.tabFocusModeOnMsg, AccessibilityHelpNLS.tabFocusModeOnMsgNoKb));
+			content.push(descriptionForCommand(ToggleTabFocusModeAction.ID, AccessibilityHelpNLS.tabFocusModeOnMsg, AccessibilityHelpNLS.tabFocusModeOnMsgNoKb, this._keybindingService));
 		} else {
-			content.push(this._descriptionForCommand(ToggleTabFocusModeAction.ID, AccessibilityHelpNLS.tabFocusModeOffMsg, AccessibilityHelpNLS.tabFocusModeOffMsgNoKb));
+			content.push(descriptionForCommand(ToggleTabFocusModeAction.ID, AccessibilityHelpNLS.tabFocusModeOffMsg, AccessibilityHelpNLS.tabFocusModeOffMsgNoKb, this._keybindingService));
 		}
 		return content.join('\n\n');
 	}
 }
```
Comment (meganrogge) [inline src/vs/workbench/contrib/accessibility/browser/accessibilityContributions.ts:112]: test2

File: src/vs/workbench/contrib/codeEditor/browser/diffEditorHelper.ts [hunk 5]
```diff
@@ -14,14 +14,15 @@ import { EmbeddedDiffEditorWidget } from 'vs/editor/browser/widget/embeddedCodeE
 import { IDiffEditorContribution } from 'vs/editor/common/editorCommon';
 import { localize } from 'vs/nls';
 import { IConfigurationService } from 'vs/platform/configuration/common/configuration';
-import { ContextKeyEqualsExpr } from 'vs/platform/contextkey/common/contextkey';
+import { ContextKeyEqualsExpr, IContextKeyService } from 'vs/platform/contextkey/common/contextkey';
```
Comment (meganrogge) [inline src/vs/workbench/contrib/codeEditor/browser/diffEditorHelper.ts:17]: test 3

**Pull request merged**